lib/game_team.rb

```diff
@@ -0,0 +1,35 @@
+class GameTeam
+  attr_reader :game_id, 
+              :team_id, 
+              :home_or_away, 
+              :game_result, 
+              :reg_or_ot, 
+              :coach, 
+              :goals, 
+              :shots, 
+              :tackles, 
+              :pim, 
+              :power_play_opp, 
+              :power_play_goals, 
+              :face_off_win_percentage, 
+              :giveaways, 
+              :takeaways
+
+  def initialize(data)
+    @game_id = data[:game_id]
+    @team_id = data[:team_id]
+    @home_or_away = data[:hoa]
+    @game_result = data[:result]
+    @reg_or_ot = data[:settled_in]
```

Comment:
I like the more specific instance var names here, it's more readable and I can understand what it's asking without having to go back to the CSV.

Reply:
Agreed with these variables - we worked through these on a paired/meeting.